db/migrate/20160501143232_add_badge_status_to_projects.rb

```diff
@@ -0,0 +1,6 @@
+class AddBadgeStatusToProjects < ActiveRecord::Migration
+  def change
+    add_column :projects, :badge_status, :string
+    add_index :projects, :badge_status
+  end
+end
```

Comment:
I've now done a version of this and tested that it rolls back and migrates forward correctly. Thanks!